sim/dex.ts

```diff
@@ -1265,7 +1288,8 @@ class ModdedDex {
 				return new TypeError(`${filePath}, if it exists, must export a non-null object`);
 			}
 			if (!dataObject[key] || typeof dataObject[key] !== 'object') {
-				return new TypeError(`${filePath}, if it exists, must export an object whose '${key}' property is a non-null object`);
+				return new TypeError(
+					`${filePath}, if it exists, must export an object whose '${key}' property is a non-null object`);
```

Comment:
(This is not exactly resolved in the sense that I would still like it reverted.)